templates/page.tpl.php

```diff
@@ -18,13 +18,9 @@
     <div class="wrapper">
 
       <div class="page-header__logo">
-        <?php if ($logo): ?>
-          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="header__logo logo" id="logo"><img src="<?php print $logo; ?>" alt="<?php print t('Home'); ?>" class="header__logo-image" /></a>
-        <?php endif; ?>
-        <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="header__logo logo" id="logo"><?php print $site_name; ?></a>
```

Comment:
Nope, I've just dropped that back in, with the ability to turn on/off site name and slogan.  Although these currently appear below the logo.

@DJMyles can you take a look at the styling here?  We need to site name and slogan to be set to a decent font size and then to sit beside the logo.